test/app.js

```diff
@@ -84,12 +85,12 @@ describe('generator-terra-module:app', function () {
       });
 
       it('fills the site examples Index file with project data', function () {
-        const index = `packages/${repository}-monster-cookies/src/terra-dev-site/doc/MonsterCookies.doc.jsx`;
+        const index = `packages/${repository}-monster-cookies/src/terra-dev-site/doc/MonsterCookies/MonsterCookies.1.doc.jsx`;
 
         assert.fileContent(index, `import React from 'react';`);
-        assert.fileContent(index, `import DefaultMonsterCookiesSrc from '!raw-loader!../../../src/terra-dev-site/doc/example/DefaultMonsterCookies.jsx';`);
-        assert.fileContent(index, `import ReadMe from '../../../docs/README.md';`);
-        assert.fileContent(index, `import { version } from '../../../package.json';`);
+        assert.fileContent(index, `import DefaultMonsterCookiesSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/DefaultMonsterCookies.jsx';`);
+        assert.fileContent(index, `import ReadMe from '../../../../docs/README.md';`);
+        assert.fileContent(index, `import { version } from '../../../../package.json';`);
         assert.fileContent(index, `export default DocPage;`);
       });
 
```

Comment:
Can we add a test case to verify the import path of the Changelog? These paths are important for correct resolution and would need updated if structure changes in the future

Reply:
With this, can we also verify the `src` prop is receiving the correct expected git url?

Reply:
Done in https://github.com/cerner/generator-terra-module/pull/100/commits/f01c02e101e519a71a66c81536b05738415dde27

Reply:
Thanks! I've ran into generator test failures by small tweaks before and was glad to have them caught in the tests!